api-flask/app/zonal_stats.py

```diff
@@ -28,39 +37,37 @@ def get_wfs_response(wfs_params):
     https://docs.geoserver.org/stable/en/user/services/wfs/reference.html
     """
     cql_filter = []
-    if "time" in wfs_params.keys():
-        from_date = datetime.strptime(wfs_params.get("time"), "%Y-%m-%d")
+    if "time" is not None:
```

Comment:
this will always be true, no? @laurentS 
did you mean wfs_params.time maybe?

Reply:
Yep, I think you're right :)